pipelines/capture__veiculo_sppo_registro_agente_verao/constants.py

```
@@ -0,0 +1,40 @@
+Valores constantes para captura de registro de agentes de verão do SPPO
+"""
+
+from datetime import datetime
+from zoneinfo import ZoneInfo
+
+from pipelines.common import constants as smtr_constants
+from pipelines.common.capture.veiculo import constants as veiculo_constants
+from pipelines.common.utils.gcp.bigquery import SourceTable
+
+TABLE_ID = "sppo_registro_agente_verao"
+SECRET_PATH = "smtr_agentes_verao"
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to: "SECRET_PATH"

(S105)